vectortile/src/main/scala/geotrellis/vectortile/VectorTile.scala

```diff
@@ -45,9 +45,9 @@ import geotrellis.util.annotations.experimental
   * @constructor This is not meant to be called directly - see this class's
   * companion object for the available helper methods.
   */
-@experimental case class VectorTile(layers: Map[String, Layer], tileExtent: Extent) {
+@experimental case class VectorTile(layers: Map[String, Layer], tileExtent: Extent, normalize: Boolean = true) {
```

Comment:
Hmmm, I'm wondering if calling this normalize is the best name for the API we want to expose, which is allowing the user to choose the polygon external winding order and defaulting to clockwise. I'm imagining a flag that's something like `windClockwise` or `winding = WindingDirection.Clockwise`. Since we only trivially get the operation for clockwise winding via the `normalize` method though, it might be extra effort to implement the CCW case which is annoying.

The docstring for `Geometry.normalize` in JTS states:
> Converts this Geometry to normal form (or canonical form). Normal form is a unique representation for Geometrys. It can be used to test whether two Geometrys are equal in a way that is independent of the ordering of the coordinates within them. Normal form equality is a stronger condition than topological equality, but weaker than pointwise equality. The definitions for normal form use the __standard lexicographical ordering for coordinates__.

(emphasis mine) which is the only place the docstring notes how this method will reorder coordinates, and I suppose in this case lexographical order means CW based on prior convention.

Given this, and that implementing CCW option would be more effort, and for now we really only care about the CW case, I'd be ok with simply adding a note to the docstring that states why this `normalize` parameter exists, what it's actually for in practice, and why you might want to turn it off, given that in practice for vector tiles I think you pretty much always want this new option set to `true`.

Reply:
Something like "force mapbox polygon winding" is what we're trying to expose in the API.  Can we come up with a name along those lines that isn't as opaque as `normalize` (which is pretty inside baseball).  `forcePolygonWinding`?

Reply:
How about `forceWindClockwise`? 
I don't think it makes sense to create a group of sealed traits only for true / false flag, so we can go with boolean.

Reply:
Agreed on boolean flag.  Clockwise winding only applies to exterior rings, so maybe not entirely appropriate.

Reply:
👍 to boolean. I'm maybe 60/40 for `forcePolygonWinding` over `forceWindClockwise` but either would be fine -- tradeoffs for both and I don't have a better, more concise name to throw in. Pick your poison!

Reply:
Okay, will try have some pretty name for this param; `normalize` is itself so confusing.